R/03-dataExplorer.R

```diff
@@ -560,7 +560,18 @@ dataExplorerServer <- function(id) {
                      "Please select a database in the sidebar panel."
                    ))
                    if(!is.na(input[["maxCharLength"]])){
-                     tabData <- cutAllLongStrings(isoDataFull(), cutAt = input[["maxCharLength"]])
+                     # convert factor to character just for table display
+                     tabData <- data.frame(lapply(isoDataFull(), function(x) {
```

Comment:
Maybe we should add the conversion of factors as option directly to the function of `cutAllLongStrings()` ... 
Since this is also an issue within the _import module_. Let us shortly discuss tomorrow, how best to fix this.